releasenotes/notes/stop-chunking-46a1a6b7f48bde14.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
[minor] Can you prefix the file `process-agent-stop-.*`? It'll make it easier to find in in the future with all the release notes that we have.